### PR TITLE
Adds Statsig to finish sign up events and a new Amp/Stat account settings page visited event

### PR DESCRIPTION
**Pull request by hannahbergam**
This PR adds Statsig logging to two finish_sign up events that are currently being sent from amplitude: the submit clicked and user type selected events. 

This also adds logging (to both Amplitude and Statsig) to the account settings page, logging when the page is visited and adding user type as a param (so we can filter on teacher). 

Screenshot 2024-04-01 at 3 49 29 PM
Screenshot 2024-04-01 at 3 52 31 PM

## Links

- spec: []()
- jira ticket: https://codedotorg.atlassian.net/browse/ACQ-1546

## PR Checklist:

- [ ] Tests provide adequate coverage
- [ ] Privacy and Security impacts have been assessed
- [ ] Code is well-commented
- [ ] New features are translatable or updates will not break translations
- [ ] Relevant documentation has been added or updated
- [ ] User impact is well-understood and desirable
- [ ] Pull Request is labeled appropriately
- [ ] Follow-up work items (including potential tech debt) are tracked and linked

### CODE REVIEW

**Review by TurnerRiley — APPROVED** [review]
LGTM!

File: apps/src/sites/studio/pages/devise/registrations/_finish_sign_up.js [hunk 1]
```diff
@@ -98,9 +98,13 @@ $(document).ready(() => {
       cleanSchoolInfo();
       $('#user_age').val('21+');
     }
-    analyticsReporter.sendEvent(EVENTS.SIGN_UP_FINISHED_EVENT, {
-      'user type': user_type,
-    });
+    analyticsReporter.sendEvent(
```
Comment (dmcavoy) [inline apps/src/sites/studio/pages/devise/registrations/_finish_sign_up.js:101]: I think this is the event we thought should maybe move to the back end? Are we going to do that in a later PR? Or add that logging in addition to this?

Comment (hannahbergam) [reply apps/src/sites/studio/pages/devise/registrations/_finish_sign_up.js:101]: I was imagining adding both! But can remove this if it would be confusing

Comment (dmcavoy) [reply apps/src/sites/studio/pages/devise/registrations/_finish_sign_up.js:101]: Lets do both for now and we can remove it if it becomes confusing and the other is working better

**Review by bethanyaconnor — APPROVED** [review]

**Pull request merged**